e2etest/bgp_test.go

```diff
@@ -927,6 +927,7 @@ var _ = ginkgo.Describe("BGP", func() {
 							Communities: []string{
 								CommunityNoAdv,
 							},
+							LocalPref: uint32Ptr(300),
```

Comment:
Are we able to verify this on the other side inspecting frr?

Reply:
I couldn’t find a show command to help me with this I just confirmed route map constructed correctly when either options or both in use it’s not always 1:1 also I wanted to mention this an ibgp only feature setting it for ebgp has no effect

Reply:
+1 to verify the local preference on the neighbor side. Here's one way to do it where the local preference is set to _300_:

```
bash-5.1# vtysh -c 'show ip bgp json'
{
 "vrfId": 0,
 "vrfName": "default",
 "tableVersion": 1,
 "routerId": "10.89.0.5",
 "defaultLocPrf": 100,
 "localAS": 64512,
 "routes": { "9.9.9.9/32": [
  {
    "valid":true,
    "bestpath":true,
    "pathFrom":"internal",
    "prefix":"9.9.9.9",
    "prefixLen":32,
    "network":"9.9.9.9\/32",
    "metric":0,
    "locPrf":300,
    "weight":0,
    "peerId":"10.89.0.3",
    "path":"",
    "origin":"IGP",
    "nexthops":[
      {
        "ip":"10.89.0.3",
        "hostname":"kind-control-plane",
        "afi":"ipv4",
        "used":true
      }
    ]
  }
] }  }
```

Could you please create another test where only local preference is set (i.e. no BGP communities)?

Reply:
added test case for localpref